docs/connecting-data/connecting-to-apis-and-external-data/servicenow-api.md

```diff
@@ -1,7 +1,164 @@
+
+### Rate Limits
+To prevent excessive inbound REST API requests, set rules that limit the number of inbound REST API requests processed per hour. 
+You can create rules to limit requests for specific users, users with specific roles, or all users. You can learn more about that in the [ServiceNow](https://docs.servicenow.com/bundle/sandiego-application-development/page/integrate/inbound-rest/concept/inbound-REST-API-rate-limiting.html){target=_blank} docs.
+
+## How to Connect DataDistillr to ServiceNow
+To set up a data source connection for ServiceNow, you will need to have:
+
+- A [unique name](#name) for your data source connection to be used in queries
+- The [Instance Name](#instance-id-username-and-password) associated ServiceNow Instance
+- The [Username](#instance-id-username-and-password) associated with your ServiceNow Instance
+- The [Password](#instance-id-username-and-password) associated with your ServiceNow Instance
+
+### Data Source Form
+To locate the ServiceNow form, follow the steps in [Connecting Your Data to DataDistillr](../../). When you get to the window to choose the data source type, select API as shown below.
+
+<figure markdown>
+  ![Data Source Wizard][image-7]{ width="100%" }
+</figure>
+
+
+
+On the API screen, select ServiceNow from the list of API forms.
+
+<figure markdown>
+  ![List of APIs][image-8]{ width="100%" }
+</figure>
+
+
+
+The following form will appear. Instructions can be found below on how to find the information required to fill each field on the ServiceNow API form.
+
+Once you have filled out all the fields, press the green 'Save' button, and your API will be connected!
+
+<figure markdown>
+  ![ServiceNow Form][image-1]{ width="100%" }
+</figure>
+
+
+### Name
+Enter any name that will help you recognize this data source from within your query window.
+
+!!! info "Acceptable Characters Include"
+
+    - lowercase alphanumeric characters
+    - underscores
+
+### Instance ID, Username and Password
+
+=== "1. Home Page"
+
+    Head over to your ServiceNow [developer portal](https://developer.servicenow.com/dev.do).
+    In the top right corner click the profile avatar.
+    
+    ![ServiceNow Home Page][image-10]
+
+=== "2. My Instance"
+
+    Click the "Manage instance password" tab.
+    
+    ![Find Instance creds][image-11]
+
+=== "3. Instance Credentials" 
+
+    Here you will find your credentials necessary (Instance name, Username and Password) to connect your ServiceNow instance to DataDistillr.
+
+    ![Instance creds][image-12]
```

Comment:
Is it possible to crop this image so it's easier to see the text - I have it on a fairly large screen and it is hard to see the instance name, username, and password text

Reply:
I tried my best